src/EFCore.PG/Extensions/NpgsqlIndexBuilderExtensions.cs

```diff
@@ -186,6 +186,16 @@ public static class NpgsqlIndexBuilderExtensions
             return indexBuilder.CanSetAnnotation(NpgsqlAnnotationNames.IndexOperators, operators, fromDataAnnotation);
         }
 
+        /// <summary>
+        /// The PostgreSQL columns transformation to 'to_tsvector()' to be used.
+        /// </summary>
+        /// <param name="indexBuilder">The builder for the index being configured.</param>
+        /// <param name="configName">to_tsvector() function config name.</param>
+        /// <returns></returns>
+        public static IndexBuilder TransformToTsVector(this IndexBuilder indexBuilder, string configName)
```

Comment:
You need to create other overloads of the this method, plus a CanSetToTsVector. See the other extension methods (e.g. HasMethod) and follow that as an example.

Reply:
Please check did I missed something?

Reply:
A generic version is missing (i.e. `UseToTsVector<TEntity>`)

Reply:
Done.